.golangci.yml

```
@@ -70,10 +70,12 @@ linters-settings:
 # Disable goconst in test files, often we have duplicated strings across tests, but don't make sense as constants.
 issues:
   exclude-rules:
-    - path: (_test\.go|utilities_testing\.go)
+    # cover utilities_testing.go and _test.go files
+    - path: (_test.*\.go)
       linters:
         - goconst
-    - path: (_test\.go|utilities_testing\.go)
+        - prealloc
+    - path: (_test.*go)
```

ditto here - sorry didn't leave suggestion in both places

 ```suggestion
    - path: (_test.*\.go)
```